soc/nxp/mcx/mcxw/soc.h

```diff
@@ -10,4 +10,5 @@
 
 #define PORT_MUX_GPIO kPORT_MuxAsGpio
 
+void soc_reset_hook(void);
```

Comment:
soc_reset_hook, i did this to call the assembly function we have instead of renaming it. this will be also a placeholder if we need to add more code in the future.
Do you want me to remove it?

Reply:
Removed. Thanks